index.js

```diff
@@ -0,0 +1 @@
+alert("Hola Piero");
```

Comment:
@Ropiex te faltó agregar una línea vacía, al final del archivo.

```suggestion
alert("Hola Piero");

```